src/ampform/dynamics/__init__.py

```diff
@@ -119,3 +121,53 @@ def formulate_form_factor(s, m_a, m_b, angular_momentum, meson_radius) -> sp.Exp
         stacklevel=1,
     )
     return FormFactor(s, m_a, m_b, angular_momentum, meson_radius)
+
+
+@unevaluated
+class DispersionIntegral(sp.Expr):
```

Comment:
We have to think about the name and location. This expression is more specific than a (once-subtracted) dispersion integral; it is the Chew–Mandelstam for any angular momentum. In that sense, it fits better under `ampform.dynamics.phasespace` with the other phase space factors.